gcloud/bigquery/table.py

```diff
@@ -419,6 +419,8 @@ def _build_resource(self):
         if self.view_query is not None:
             view = resource['view'] = {}
             view['query'] = self.view_query
+            # Back-end rejects 'schema' when passing a query.
+            del resource['schema']
```

Comment:
@tseaver Why not

```python
if self.view_query is not None:
     view = resource['view'] = {}
     view['query'] = self.view_query
else:
     # Schema can only be used on non-queries.
     resource['schema'] = {'fields': _build_schema_resource(self._schema)}
```

and then just drop out the `schema` from `resource` above. Seems a bit silly to add and `del` when we could do this and avoid instead.